src/applications/disability-benefits/781/helpers.js

```diff
@@ -1,33 +1,99 @@
 import React from 'react';
 import AdditionalInfo from '@department-of-veterans-affairs/formation/AdditionalInfo';
 
-export const introductionText = (
+let incidentTitle;
+let incidentText;
+
+const getPtsdClassification781 = formData => {
+  const classifications = formData['view:selectablePtsdTypes'];
```

Comment:
You should be able to combine this and `getPtsdClassification781a` into 1 function.

Reply:
I tried to combine getPtsdClassification781 with getPtsdClassificaiton781a. The problem was that if the user chooses all 4 types, I couldn't figure out a way for the correct PTSD types to display on the correct flows.

For example: Combat and Non-combat on 781 and Personal Assault and Military Sexual Assault on 781a. 

I separated them out for the two different flows: 781 and 781a. If you have a suggestion for how to solve this, please let me know. It looks ridiculous, but I'm limited on what I can do with the form structure. Normally, I would not willingly turn something in that looks like this. 

@robfelder @igreulich

Reply:
Never mind! I get it know. The switch statement will fix this problem. Thank you!

Reply:
This may sound kinda crazy, but let me know what you think. What do you think is the probability of us needing to find out what form we're on? I'm wondering if a function could be used that determines what form is currently being worked based off `view:selectablePtsdTypes`. It could return info such as if it's 781 Combat, 781 Non-Combat, 781 Both, etc. Idk, just a thought.

Reply:
Yes! This sounds like a good idea. Let me see if I can work on a function that will determine the form.
@aaronjlech : What are your thoughts?

Reply:
@imeecuison It's a good idea, but we're already splitting the code as of now for each form section(IE: 2 pages for event date, 1 for 781a, 1 for 781)